tools/cold-backup/templates/patch.json

```diff
@@ -2,8 +2,9 @@
     "spec": {
         "containers": [
             {
-                "name": "dummy",
-                "image": "centos:7",
+                "name": "strimzi-backup",
+                "image": "registry.access.redhat.com/ubi8/ubi:8.4",
```

Comment:
Would `ubi-minimal` image be sufficient? Or does it need the regular `ubi` image?

Reply:
No. There is no tar package in there.